packages/fx-core/templates/plugins/resource/hosting/bicep/function/config.template.bicep

```diff
@@ -1,5 +1,5 @@
 // Get existing app settings for merge
-var functionCurrentAppSettings = list('${provisionOutputs.functionOutput.value.resourceId}/config/appsettings', '2021-02-01').properties
+var functionCurrentAppSettings = list('${provisionOutputs.functionOutput.value.botWebAppResourceId}/config/appsettings', '2021-02-01').properties
```

Comment:
Since it is Azure functions bicep file, why it contains "WebApp"? 
Another question is why we change the name of "resourceId"?